pkg/items/suggest_item.go

```diff
@@ -0,0 +1,8 @@
+package items
+
+// SuggestItem represents a suggestion with distance and count (placeholder).
+type SuggestItem struct {
```

Comment:
I'm coming a bit late as I only started to look at your repository a few hours ago.

I'm interesting in ypur project and I would like to provide you feedbacks.

I can help by opening PR, reviews but also setting up some CI things.

Here the feedback I would have made would be one revive.var-naming could have done.


```suggestion
// Suggestion represents a suggestion with distance and count (placeholder).
type Suggestion struct {
```

The reason here is the fact that if you look at this package everything is fine, but if you look a bit further you will see that outside the package you are now using `items.SuggestItem`, item is repeated. 

By using `Suggestion` is package `items`, your code would have called `items.Suggestion`.

This is the kind of feedbacks I could do.

So please tell me if you are open to reviews and suggestions

I can help you will setting up CI to detect many things with golangci-lint

Reply:
Hi @ccoVeille,

Thank you for taking the time to look into the repository and for your valuable feedback! I really appreciate your interest and your suggestions regarding naming conventions and CI improvements.

Your point about SuggestItem vs. Suggestion makes a lot of sense, and I agree that a clearer naming strategy would improve readability and maintainability. I’m definitely open to your reviews and contributions, so feel free to open PRs with your suggestions!

Setting up golangci-lint to catch issues like these automatically would also be a great improvement. If you’re interested in helping with that, I’d be happy to collaborate.

Looking forward to your contributions! 🚀

Thanks again!

Reply:
Here we go

- https://github.com/snapp-incubator/go-symspell/pull/6
- https://github.com/snapp-incubator/go-symspell/pull/7
- https://github.com/snapp-incubator/go-symspell/pull/8